yarn-project/circuits.js/src/structs/contract_storage_read.ts

```diff
@@ -21,6 +21,7 @@ export class ContractStorageRead {
      */
     public readonly currentValue: Fr,
     /**
+     * TODO: WILL NEED THIS for vm? why are these not serialised?
```

Comment:
We are using counter in hints for storage reads

Reply:
so we will need to update this?

Reply:
if so ill do so in a followup